mllib/src/main/scala/org/apache/spark/ml/regression/DecisionTreeRegressor.scala

```diff
@@ -198,7 +198,8 @@ class DecisionTreeRegressionModel private[ml] (
   }
 
   /** We need to update this function if we ever add other impurity measures. */
-  protected def predictVariance(features: Vector): Double = {
+  @Since("3.0.0")
+  def predictVariance(features: Vector): Double = {
```

Comment:
This is the only one I'm not so sure about. I don't know if it's that useful or if it's well defined ... it's not a variance but some impurity measure, which I suppose happens to be variance in the regressor's prediction at the moment. Not super against it, but is it useful to you?

Reply:
You are right. I think we do not need this method for now. I will remove it.